client/views/chat.tpl

```diff
@@ -2,7 +2,11 @@
 <div id="chan-{{id}}" data-title="{{name}}" data-id="{{id}}" data-type="{{type}}" class="chan {{type}}">
 	<div class="header">
 		<button class="lt"></button>
-		<button class="rt"></button>
+		{{#equal type "lobby"}}
+			{{!-- Insert settings icon --}}
+		{{else}}
+			<button class="rt"></button>
+		{{/equal}}
```

Comment:
I'm very not fond of placeholders like this one for potential future code/UI elements that might not be located there. Could you instead check that the type is _not_ lobby then display the button, without `else` statement please?